docs/01_nodeos/03_plugins/trace_api_plugin/index.md

```diff
@@ -1,8 +1,20 @@
 # trace_api_plugin
 
-## Description
+## Overview
 
-The `trace_api_plugin` provides a consumer-focused long-term API for retrieving retired actions and related metadata from a specified block. The plugin defines a new HTTP endpoint accessible (see the [API reference](api-reference/index.md) for more information).
+The `trace_api_plugin` provides a consumer-focused long-term API for retrieving retired actions and related metadata from a specified block. The plugin defines a new HTTP endpoint accessible at `/v1/trace_api/get_block` by the **POST** method and stores serialized block trace data to the filestream for later retrieval via HTTP RPC requests.
+
+For reference information, see the [Trace API reference documentation](api-reference/index.md).
+
+## Purpose
+
+While integrating applications such as block explorers and exchanges with an EOSIO blockchain, the user might require a complete transcript of actions that are processed by the blockchain, including those spawned from the execution of smart contracts and scheduled transactions. The `trace_api_plugin` aims to serve this need. The purpose of the plugin is to provide:
```

Comment:
simpler constructions are suggested:
"actions that are processed" --> "actions processed"
"The `trace_api_plugin` aims to serve this need" --> "The `trace_api_plugin` serves this need"

Reply:
Applied